README.md

```diff
@@ -80,7 +80,7 @@ If you have used pycytominer in your project, please cite us as:
 
 ```
 @software{pycytominer,
-  author = {Way, G.P., Chandrasekaran, S.N., Bornholdt, M., Tsang, H., Adeboye, A., Cimini, B., Weisbart, E., Ryder, P., Stirling, D., Jamali, N., Carpenter, A.E., Singh, S.},
+  author = {Way, G.P., Chandrasekaran, S.N., Bornholdt, M., Tsang, H., Adeboye, A., Cimini, B., Weisbart, E., Ryder, P., Stirling, D., Jamali, N., Fleming, S.J., Carpenter, A.E., Singh, S.},
```

Comment:
```suggestion
  author = {Way, G.P., Chandrasekaran, S.N., Bornholdt, M., Fleming, S.J., Tsang, H., Adeboye, A., Cimini, B., Weisbart, E., Ryder, P., Stirling, D., Jamali, N., Carpenter, A.E., Singh, S.},
```

Thanks Stephen! I moved you based on lines contributed - a poor metric, I know, but one we have right now. We do have desires to create a more equitable and representative solution, however (discussed in #158 )

Reply:
Fine by me!  Happy to be included :)